src/components/ComboBox.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Should ComboBox be part of the translation-helps-rcl, specifically the card or card settings component?